__tests__/unit/client.test.ts

```diff
@@ -1,11 +1,35 @@
-import { Client, ClientClosedError, fql, NodeHTTP2Client } from "../../src";
+import {
+  Client,
+  ClientClosedError,
+  QueryRuntimeError,
+  fql,
+  NodeHTTP2Client,
+} from "../../src";
 import {
   getClient,
   getDefaultHTTPClientOptions,
   getDefaultSecretAndEndpoint,
 } from "../client";
 
 describe("Client", () => {
+  it("Allows setting a secret in query", async () => {
+    const client = getClient();
```

Comment:
Make sure the test fails if a different error occurs.

```suggestion
  it("Allows setting a secret in query", async () => {
    expect.assertions(1)
  
    const client = getClient();
```